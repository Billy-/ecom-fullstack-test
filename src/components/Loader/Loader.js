import React from 'react';

// TODO: Shiny animated spinning loader :D
const Loader = () => (
    <h1>Loading...</h1>
);

export default Loader;
