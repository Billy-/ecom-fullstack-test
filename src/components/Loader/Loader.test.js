import React from 'react';
import renderer from 'react-test-renderer';

import Loader from './Loader';

const props = {};

describe('Loader', () => {
    it('renders without throwing', () => {
        renderer.create(<Loader {...props} />);
    });

    it('matches snapshot', () => {
        const tree = renderer
            .create(<Loader {...props} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
