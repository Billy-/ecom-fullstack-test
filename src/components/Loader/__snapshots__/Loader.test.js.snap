// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader matches snapshot 1`] = `
<h1>
  Loading...
</h1>
`;
